extensions/infobip/v1/actions/sendSms/config/fields.ts

```diff
@@ -0,0 +1,42 @@
+import { z, type ZodTypeAny } from 'zod'
+import {
+  type Field,
+  FieldType,
+  StringType,
+  makeStringOptional,
+} from '@awell-health/extensions-core'
+
+export const fields = {
+  from: {
+    label: 'From',
+    id: 'from',
+    type: FieldType.STRING,
+    stringType: StringType.PHONE,
+    required: false,
+    description:
+      'The phone number you wish to use for sending the text message. Defaults to value provided in settings.',
+  },
+  to: {
+    label: 'To',
+    id: 'to',
+    type: FieldType.STRING,
+    stringType: StringType.PHONE,
+    required: true,
+    description:
+      'The phone number to which you intend to send the text message',
+  },
+  text: {
+    label: 'Text',
+    id: 'text',
```

Comment:
I see the API uses text, but perhaps we should use something easier to understand as the label at least

Infobip API docs themselves describe this field as "Content of the message being sent."